lib/pf/web/guest.pm

```diff
@@ -490,6 +491,7 @@ Readonly::Hash our %ERRORS => (
     $ERROR_SPONSOR_NOT_ALLOWED  => 'Email %s is not allowed to sponsor guest access',
     $ERROR_PREREG_NOT_ALLOWED  => 'Guest pre-registration is not allowed by policy',
     $ERROR_INVALID_PIN => 'PIN is Invalid!',
+    $ERROR_MAX_RETRIES => 'Maximum amount of retries attempted',
```

Comment:
Please update `conf/locale/en/LC_MESSAGES/packetfence.po`. I will eventually extend `extract_i18n_strings.pl` to cover the captive portal and not only the Web admin.